.snippets/code/tutorials/eth-api/foundry-start-to-end/terminal/test.md

```diff
@@ -0,0 +1,16 @@
+<div id="termynal" data-termynal>
+    <span data-ty="input">foundry % forge test</span>
```

Comment:
```suggestion
    <span data-ty="input">forge test</span>
```

Reply:
actually, all of these are missing the standard file path, can you please make sure that they're all using 

```
<span class="file-path"></span>
```